WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php

```diff
@@ -123,7 +123,9 @@ public function process( File $phpcsFile, $stackPtr ) {
 				return;
 			}
 
-			if ( 1 === preg_match( '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $tokens[ $nextToken ]['content'] ) ) {
+			$nextNextToken = $phpcsFile->findNext( Tokens::$emptyTokens, ( $nextToken + 1 ), null, true, null, true );
```

Comment:
Instead of searching for the next next token, could we see if the current next token is a function call (type is `T_FUNCTION` or something?), and skip over it that way?

Reply:
Next best thing to do would be to add `T_COMMENT` as part of what not to look for I think.

Reply:
Maybe abstract from WPCS's AbstractFunctionRestrictions, and then use the method [here](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/AbstractFunctionRestrictionsSniff.php#L205) ?

Reply:
I can do that but that would still end up returning `true` for constants it looks like, unless I'm _doing_it_wrong.

Reply:
I can do that but that would still end up returning `true` for constants it looks like, so we'd still need to search for the nextNextToken.